Dockerfile

```diff
@@ -9,6 +9,10 @@ RUN apk upgrade
 # Install git so we can use it to grab Terraform modules
 RUN apk add --update git
 
+# Enable re-templates of Terraform Code
+# Reference: https://github.com/GSA/data.gov/issues/3083
+ENV BROKERPAK_UPDATES_ENABLED true
```

Comment:
Env vars related to how it gets configured should probably happen on the `docker compose` side, but since this is presumably going to come out from behind the feature-flag I'm OK with doing it here, so that for the consumers of the image, it's as if that already happened.

HOWEVER, remember that the production SSB deployment doesn't use this image, so we need to also set it up in the app deployments.

Reply:
Good point.  I thought it was going to be a requirement for our operations for the foreseeable future, so it aligned with our other GSA customizations.  But I agree that this is not a catch-all for our apps (unfortunately).